code/_onclick/hud/screen_objects/base_screen_objects.dm

```diff
@@ -372,6 +381,10 @@
 //--------------------------------------------------sanity---------------------------------------------------------
 /obj/screen/sanity
 	name = "sanity"
+	desc = "Soundness of your mind. Not keeping it in check may result in a breakdown.\
```

Comment:
Just clicking the sanity element brings up the informative menu, this should be mention in this description